packages/core/src/providers/clerk.ts

```diff
@@ -0,0 +1,117 @@
+/**
+ * <div style={{display: "flex", justifyContent: "space-between", alignItems: "center"}}>
+ * <span style={{fontSize: "1.35rem" }}>
+ *  Built-in sign in with <b>Clerk</b> integration.
+ * </span>
+ * <a href="https://clerk.com" style={{backgroundColor: "black", padding: "12px", borderRadius: "100%" }}>
+ *   <img style={{display: "block"}} src="https://authjs.dev/img/providers/clerk.svg" width="24"/>
+ * </a>
+ * </div>
+ *
+ * @module providers/clerk
+ */
+import type { OAuthConfig, OAuthUserConfig } from "./index.js"
+
+/** @see [Get the authenticated user](https://clerk.com/docs/reference/frontend-api/tag/OAuth2-Identify-Provider#operation/getOAuthUserInfo). */
+export interface ClerkProfile {
+  object: "string",
+  instance_id: "string",
+  email: "string",
+  email_verified: boolean,
+  family_name: "string",
+  given_name: "string",
+  name: "string",
+  username: "string",
+  picture: "string",
+  user_id: "string",
+  public_metadata?: any,
+  private_metadata?: any,
+  unsafe_metadata?: any
+}
+
+/**
+ * Add Clerk login to your page.
+ *
+ * ### Setup
+ *
+ * #### Callback URL
+ * ```
+ * https://example.com/api/auth/callback/clerk
+ * ```
+ *
+ * #### Configuration
+ * ```ts
+ * import { Auth } from "@auth/core"
+ * import Clerk from "@auth/core/providers/clerk"
+ *
+ * const request = new Request(origin)
+ * const response = await Auth(request, {
+ *   providers: [
+ *     Clerk({ clientId: CLERK_CLIENT_ID, clientSecret: CLERK_CLIENT_SECRET, baseUrl: CLERK_BASE_URL }),
```

Comment:
`pnpm format`

```suggestion
 *     Clerk({
 *       clientId: CLERK_CLIENT_ID,
 *       clientSecret: CLERK_CLIENT_SECRET,
 *       baseUrl: CLERK_BASE_URL,
 *     }),
```

Reply:
Thanks @k-taro56, I run `pnpm format` and several changes have been applied to the Clerk provider file: https://github.com/nextauthjs/next-auth/pull/11349/commits/26a74589d1f38a8e46ae34a449e542d8b61b5246